src/lsm/forest_fuzz.zig

```diff
@@ -370,8 +498,13 @@ pub fn generate_fuzz_ops(random: std.rand.Random, fuzz_op_count: usize) ![]const
                     // Can only checkpoint on the last beat of the bar.
                     compact_op % constants.lsm_batch_multiple == constants.lsm_batch_multiple - 1 and
                     compact_op > constants.lsm_batch_multiple and
+                    // Never checkpoint at the same op twice
+                    compact_op > checkpointed_op + constants.lsm_batch_multiple and
```

Comment:
Curious - do we need `compact_op > checkpointed_op + constants.lsm_batch_multiple` or is `compact_op > checkpointed_op` sufficient? 

`compact_op % constants.lsm_batch_multiple == constants.lsm_batch_multiple - 1` allows checkpoints to be one measure apart, but `compact_op > checkpointed_op + constants.lsm_batch_multiple` requires them to be at least two measures apart. Is that intentional?

Reply:
I think it's correct as it is. Currently, `checkpointed_op` is the highest op number that has made it to disk, not the op number when the checkpoint actually happened (which is one bar ahead). So it's actually not two measures apart, only one.

Thinking about it, the naming is clearly confusing. I'll rename it to `persisted_op`.